libs/auth-api-lib/src/lib/clients/admin/dto/admin-client.dto.ts

```diff
@@ -101,4 +102,9 @@ export class AdminClientDto {
     type: [AdminClientClaimDto],
   })
   customClaims?: AdminClientClaimDto[]
+
+  @ApiProperty({
+    description: 'Allowed scopes for the client',
+  })
+  allowedScopes?: Pick<ClientAllowedScope, 'scopeName' | 'clientId'>[]
```

Comment:
This unfortunately doesn't generate the correct openapi schema, which won't generate the correct signature on the client function in the GQL layer.
I see that we have already `ClientAllowedScopeDTO` that we can just reuse here:

```suggestion
  @ApiProperty({
    description: 'Allowed scopes for the client',
    type: [ClientAllowedScopeDTO]
  })
  allowedScopes?: ClientAllowedScopeDTO[]
```

Reply:
Are you talking about the Pick or that I have to add the type?